bin/docker-image-tool.sh

```diff
@@ -110,7 +110,7 @@ function build {
         error "Cannot found dir SPARK_HOME $SPARK_HOME, you must configure SPARK_HOME correct."
       fi
     fi
-    cp -r "$SPARK_HOME/" "$KYUUBI_ROOT/spark-binary/"
+    cp -r "$SPARK_HOME/." "$KYUUBI_ROOT/spark-binary/"
```

Comment:
does `"$SPARK_HOME/*"` work? it's a more common case than that dot

Reply:
> does `"$SPARK_HOME/*"` work? it's a more common case than that dot

oh... `*` does not work. It throws:

```
cp: cannot stat '/data00/home/yanghua/kyuubi/dist/externals/spark-3.4.2-bin-hadoop3/*': No such file or directory
```